homeassistant/components/velbus/light.py

```diff
@@ -96,6 +96,7 @@ class VelbusButtonLight(VelbusEntity, LightEntity):
 
     _channel: VelbusButton
     _attr_entity_registry_enabled_default = False
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
```

Comment:
Now that they are diagnostic, do they still need to be disabled by default?

Reply:
for some modules there can be up to 64 of these entities, so yes i would keep them disabled by default

Reply:
Hehe ok, that makes sense 🤣

Reply:
Shouldn't these be config category?

Reply:
Oh right 🤦  Goodmorning @MartinHjelmare! Always good to have you around :D

Reply:
#66156

Reply:
why config instead of diagnostics?

Reply:
Diagnostic category is only for read only entities like sensors and binary sensors.

https://developers.home-assistant.io/docs/core/entity#generic-properties

Reply:
ok clear, thanks for the explanation